tests/conftest.py

```diff
@@ -34,4 +34,20 @@ def runner(app):
     """Using test app, create and return a CLI runner object."""
 
     return app.test_cli_runner()
+class AuthActions(object):
+    def __init__(self, client):
+        self._client = client
+
+    def login(self, email='doo@stevenscollege.edu', password='snacks'):
```

Comment:
we should maybe use some of the mock data instead of a clients actual email and password? i thin your test will work perfectly, but maybe change to "teacher@stevenscollege.edu" and "qwerty"